CustomItems/Items/Scp5355.cs

```diff
@@ -0,0 +1,168 @@
+        public float SizeChange { get; set; } = 0.1f;
+
+        /// <inheritdoc/>
+        protected override void SubscribeEvents()
+        {
+            Exiled.Events.Handlers.Player.Destroying += OnDestroying;
+            Exiled.Events.Handlers.Player.Died += OnDied;
+
+            base.SubscribeEvents();
+        }
+
+        /// <inheritdoc/>
+        protected override void UnsubscribeEvents()
+        {
+            Exiled.Events.Handlers.Player.Destroying -= OnDestroying;
+            Exiled.Events.Handlers.Player.Died -= OnDied;
+
+            base.UnsubscribeEvents();
+        }
+
+        /// <inheritdoc/>
+        protected override void OnReloading(ReloadingWeaponEventArgs ev)
+        {
+            if (ev.IsAnimationOnly)
+            {
+                mode++;
+                string modeText;
+                switch (mode)
+                {
+                    case 1:
+                        modeText = "Shrink";
+                        break;
+                    case 2:
+                        modeText = "Enlarge";
+                        break;
+                    case 3:
+                        modeText = "Return";
+                        break;
+                    default:
+                        mode = 1;
+                        modeText = "Shrink";
+                        break;
+                }
+                ev.Player.ReferenceHub.hints.Show(new TextHint($"{modeText}", new HintParameter[]
+                    { new StringHintParameter("") }, HintEffectPresets.FadeInAndOut(15f, 1f, 15f), 15f));
+            }
+        }
+
+
+        /// <inheritdoc/>
+        protected override void OnShooting(ShootingEventArgs ev)
+        {
+            base.OnShooting(ev);
+            ev.Shooter.SetWeaponAmmo(ev.Shooter.CurrentItem, (int)ClipSize - 1);
+            Player Target = Player.Get(ev.Target);
+            if (Target != null)
+            {
+                if (!sizedPlayers.ContainsKey(Target))
+                {
+                    sizedPlayers.Add(Target, Target.Scale);
+                }
+
+                Vector3 NewSize = Target.Scale;
+                switch (mode)
+                {
+                    case 1:
+                        NewSize -= new Vector3(SizeChange, SizeChange, SizeChange);
+                        if (NewSize.x < .4)
+                            NewSize = new Vector3(.4f, .4f, .4f);
+                        break;
+                    case 2:
+                        NewSize += new Vector3(SizeChange, SizeChange, SizeChange);
+                        if (NewSize.x > 1.5)
+                            NewSize = new Vector3(1.5f, 1.5f, 1.5f);
+                        break;
+                    case 3:
+                        if (sizedPlayers.ContainsKey(Target))
+                        {
+                            NewSize = sizedPlayers[Target];
+                        }
+
+                        break;
+                }
+
+                Target.Scale = NewSize;
```

Comment:
In my opinion, you should check if Target is at minimum/maximum scale so it won't be respawned unnecessarily with the same scale which makes a Target T-Pose for a brief moment